### PR TITLE
closes #58 added warn.conflict to a all files

### LINKED ISSUE #58: Documentation:  Suppress Package Messages in examples
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

https://pharmaverse.github.io/admiralpeds/dev/reference/derive_params_growth_age.html

image

### PR DESCRIPTION
**Pull request by santarabantoosoo**
According to the [references](https://pharmaverse.github.io/admiralpeds/dev/reference/index.html) page, I found only two user-facing functions for examples. I have suppressed warning for a single function [derive_params_growth_age()](https://pharmaverse.github.io/admiralpeds/dev/reference/derive_params_growth_age.html) only as only this one was showing warning messages in the vignette. Would you like me to change the other function as well? 

Also, do I need to add the suppress message function to internal functions that are not in the vignette? 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: R/derive_params_growth_age.R
Comment (rossfarrugia) [inline R/derive_params_growth_age.R:103]: @santarabantoosoo i checked a few admiral core functions and there they seem to use `library(dplyr, warn.conflicts = FALSE)` - can you double check this approach works ok and if so match it here.

I would add same to https://github.com/pharmaverse/admiralpeds/blob/main/R/derive_params_growth_height.R for consistency too.

Also feel free to add yourself to the DESCRIPTION file as a contributor as part of this PR.

**Review by rossfarrugia — CHANGES_REQUESTED** [review]

File: R/derive_params_growth_age.R
Comment (santarabantoosoo) [reply R/derive_params_growth_age.R]: Thanks Ross. 

 I added warn.conflicts = FALSE to both files and added my name as a contributor.

File: DESCRIPTION
Comment (rossfarrugia) [inline DESCRIPTION:20]: thanks @santarabantoosoo - one final minor comment: can you move this up above the 3 company names (as we usually have the funders come last) and you can take out your email address - we usually only add this for the maintainer. once done and man file updated then I'll approve and merge this one

Comment (santarabantoosoo) [reply DESCRIPTION]: Done 

Thanks Ross

**Review by rossfarrugia — APPROVED** [review]

**Pull request merged**
